AegeanTools/fitting.py

```diff
@@ -1187,7 +1196,7 @@ def new_errors(source, model, wcshelper):  # pragma: no cover
 
     return source
 
-
+# @njit # TODO explore how we can make Numba work with this
```

Comment:
This and all other commented @njit sections should be removed. I consider the TODO to be resolved as you've already "explored".

Reply:
They have been removed.